crates/autopilot/src/arguments.rs

```diff
@@ -225,79 +225,129 @@ pub struct Arguments {
 
 impl std::fmt::Display for Arguments {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}", self.shared)?;
-        write!(f, "{}", self.order_quoting)?;
-        write!(f, "{}", self.http_client)?;
-        write!(f, "{}", self.token_owner_finder)?;
-        write!(f, "{}", self.price_estimation)?;
-        display_option(f, "tracing_node_url", &self.tracing_node_url)?;
-        writeln!(f, "ethflow_contract: {:?}", self.ethflow_contract)?;
-        writeln!(
-            f,
-            "ethflow_indexing_start: {:?}",
-            self.ethflow_indexing_start
-        )?;
-        writeln!(f, "metrics_address: {}", self.metrics_address)?;
+        let Arguments {
+            shared,
+            order_quoting,
+            http_client,
+            token_owner_finder,
+            price_estimation,
+            tracing_node_url,
+            ethflow_contract,
+            ethflow_indexing_start,
+            metrics_address,
+            skip_event_sync,
+            allowed_tokens,
+            unsupported_tokens,
+            token_quality_cache_expiry,
+            pool_cache_lru_size,
+            native_price_estimators,
+            min_order_validity_period,
+            banned_users,
+            max_auction_age,
+            limit_order_price_factor,
+            fee_objective_scaling_factor,
+            trusted_tokens_url,
+            trusted_tokens,
+            trusted_tokens_update_interval,
+            enable_colocation,
+            drivers,
+            submission_deadline,
+            additional_deadline_for_rewards,
+            score_cap,
+            shadow,
+            solve_deadline,
+            fee_policy,
+            order_events_cleanup_interval,
+            order_events_cleanup_threshold,
+            db_url,
```

Comment:
What a big brain move! 😄 
BTW if you prefer you can ignore the DB url right inside the destructured `Arguments`.
```suggestion
            db_url: _intentionally_ignored_db_url,
```

Reply:
Nice!
Thinking about it, I prefer it as it is right now because of the line `writeln!(f, "db_url: SECRET")?;`: if we ignore the database near that line, I hope we remember to edit this line if we edit `db_url`.